pkg/webui/console/components/user-data-form/index.js

```diff
@@ -179,29 +223,55 @@ class UserForm extends React.Component {
           name="state"
           component={Select}
           options={approvalStateOptions}
+          required
         />
         <Form.Field
           title={sharedMessages.admin}
           name="admin"
           component={Checkbox}
           label={m.adminLabel}
         />
+        {!update && (
+          <Form.Field
+            title={sharedMessages.password}
+            component={Input}
+            name="password"
+            type="password"
+            autoComplete="new-password"
+            required
+          />
+        )}
+        {!update && (
+          <Form.Field
+            title={sharedMessages.confirmPassword}
+            component={Input}
+            name="confirmPassword"
+            type="password"
+            autoComplete="new-password"
+            required
+          />
+        )}
```

Comment:
```suggestion
        {!update && (
          <Form.Field
            title={sharedMessages.password}
            component={Input}
            name="password"
            type="password"
            autoComplete="new-password"
            required
          />
          <Form.Field
            title={sharedMessages.confirmPassword}
            component={Input}
            name="confirmPassword"
            type="password"
            autoComplete="new-password"
            required
          />
        )}
```

Reply:
this doesn't work unless I wrap the JSX in fragment tags. Is that better?

Reply:
Yeah indeed, forgot to add that. Use `<> … </>`